frontend/src/views/SecretMainPage/components/SecretImportListView/SecretImportItem.tsx

```diff
@@ -77,6 +94,28 @@ export const SecretImportItem = ({
     transition
   };
 
+  const handleResyncSecretReplication = async () => {
+    if (resyncSecretReplication.isLoading) return;
+    try {
+      await resyncSecretReplication.mutateAsync({
+        id,
+        environment,
+        path: secretPath,
+        projectId: currentWorkspace?.id || ""
+      });
+      createNotification({
+        text: "Kindly refresh the board to see changes.",
+        type: "success"
+      });
+    } catch (error) {
+      console.error(error);
+      createNotification({
+        text: "Failed to resync replication",
+        type: "error"
+      });
+    }
+  };
```

Comment:
for this one, can we do the refetch behind the scenes instead of prompting to user to explicitly refresh?

Reply:
We cannot do that, because it will affect the edited secrets in dashboard